### PR TITLE
[docs] auth docs

### PR DESCRIPTION
**Pull request by ccatherinetan**
[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

### :palm_tree: What's new in this PR
[//]: # "REQUIRED - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
* IN PROGRESS! Just wanted to put this up in case I can't finish, so someone can build on top of this. Also to get feedback early! 

### :evergreen_tree: Screenshots
[//]: # "REQUIRED for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! If you are making changes here, please CC: @kyrenetam at the bottom."

## :deciduous_tree: How to review
[//]: # 'REQUIRED - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'

## :seedling: Next steps 
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

### :information_source: Online sources
[//]: # 'Optional - copy links to any tutorials or documentation that was useful to you when working on this PR'

### :potted_plant: Related PRs
[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

[//]: # 'This tags the project leader as a default. Feel free to change, or add on anyone who you should be in on the conversation.'
CC: @varortz

### CODE REVIEW

File: docs/dev/app/auth.md [hunk 1]
```diff
@@ -4,10 +4,95 @@ prev: /dev/app/
...
+const auth = useAuth();
+```
+
 ## Sign Up Flow
+
+New users creating an account for the first time will go through the sign-up flow, handled in `src/app/(auth)/signup`. 
+1. Users are prompted to create a new account with an email and password. 
+
+Edge Cases 
+- Attempting to create a new account with an existing user's email: we do not specifically prevent users from creating a new
```
Comment (ccatherinetan) [inline docs/dev/app/auth.md]: how is this case handled again?

File: docs/dev/app/auth.md [hunk 2]
```diff
@@ -4,10 +4,95 @@ prev: /dev/app/
 
 # Authentication
 
-Basic structure; feel free to deviate.
+## User Auth Flows 
+
+**Log In**
```
Comment (ijp-dev) [inline docs/dev/app/auth.md:9]: could we make this a `###` header? I think that helps it stand out more as a subsection of the User Auth Flows section

File: docs/dev/app/auth.md (hunk 1 above)
Comment (varortz) [reply docs/dev/app/auth.md]: Currently, users cannot sign up with an email that is already registered, and they're given a general ("Something went wrong. Please try again.") error message when attempting to do so.

File: docs/dev/app/auth.md [hunk 3]
```diff
@@ -4,10 +4,107 @@ prev: /dev/app/
...
+
+New users creating an account for the first time will go through the sign-up flow, handled in `src/app/(auth)/signup`. Users can also navigate from `/signup` to the login flow if they already have an account. 
+
+1. Users are prompted to create a new account with an email and password. 
+2. If the new account credentials are valid, the user is directed to the `/verification-email-sent` page, indicating that an account-verification email has been sent to them. The user can also choose to resend the email on this page, if they think there has been an error. 
+3. After the user clicks on the verification-link sent via email, the user is directed to `/email-verified`. From this page, users are directed to navigate back to Onboarding.  
+
+Edge Cases 
+- Currently, users cannot sign up with an email that is already registered, and they're given a general ("Something went wrong. Please try again.") error message when attempting to do so.
+- Accounts that have not been verified are/are not allowed to login regularly and continue to onboarding.
```
Comment (ccatherinetan) [inline docs/dev/app/auth.md:32]: Does account verification affect/limit a user's actions in any way?
